utilities/vcf2json.py

```diff
@@ -126,6 +127,12 @@ def vcf2json(vcf_filename=None, ref_build=None, patient_id=None,
     if not patient_id:
         patient_id = vcf_reader.samples[sample_position]
 
+    codeDict = {}
+    #Read in code table for molecular consequence data. System should be in first column, code should be in second, and the display
+    #consequence name should be in the third column.
+    with open('SOCodeTable.csv') as codeTable:
+        reader = csv.reader(codeTable)
+        codeDict = {rows[2]:[rows[0], rows[1]] for rows in reader}
```

Comment:
Can we have this defined in the [common.py](https://github.com/FHIR/genomics-operations/blob/main/utilities/common.py) under utilities file like [this](https://github.com/FHIR/genomics-operations/blob/main/app/common.py#L41).